syft/generic/object.py

```diff
@@ -35,10 +35,7 @@ def __init__(
                 a chain of tensors
         """
         self.owner = owner
-        if id is None:
-            self.id = sy.ID_PROVIDER.pop()
-        else:
-            self.id = id
+        self.id = id or sy.ID_PROVIDER.pop()
```

Comment:
could we make this a bit more verbose to be `id if id else sy.ID_PROVIDER.pop()` to me that is much more clear than or but still is a 1 liner and should be the same speed